lib/child_process.js

```diff
@@ -195,6 +195,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   var ex = null;
 
+  var cmd = file;
+
```

Comment:
I believe the whole `arguments[..]` and named parameter mixed usage deopt has been around for awhile, long before V8 5.6.